### ISSUE TITLE
Health monitoring, journaling

### ISSUE THREAD

**Issue by 130s** [130s/compenv_ubuntu#122]
Would be nice to track health condition. Semi-/Automatically being able to keep track of condistion helps instantly and also for long-term. Requirement:
- tracking over the time:
  - temperature . Only when feel like I like to measure the temp (usually when not feeling well).
  - heartbeat
  - weight --> already done by Withing Scale for 4 years :)
- diary, take note. Record something not quantitative, maybe manually.
- Accessible from both Android and Web. Preferably iOS too.
  - Limiting to Android makes harder to:
    - record a good amount of input.
    - share with family who doesn't use Android.
- Data exportable...Not sure how long a particular tool/sevice lasts.

Integrate the above into tool/service @130s has already been using is the best, to reduce tools to use on a regular basis. That makes me wonder if there are `Withing`-capable apps already there to satisfy above. So I looked at [Works with Withings](https://www.withings.com/us/en/with-withings) page but didn't find anything that seems to work perfectly.

Tools:
- [FitTrend](https://www.fittrend.com/). Looks not frequently updated (last update 201609, only once in 2016?) makes me worry. -1
- [Foodzy](https://foodzy.com). Some links on their website is not working. Android app last updated May 14, 2015, [review rate is only 2](https://play.google.com/store/apps/details?id=com.foodzy&hl=en).
- [hCG Diet Life](https://play.google.com/store/apps/details?id=com.nutrisoft.hcglife). Android app review rate is only 2.5. Last update July 6, 2015
- [Withings Thermo](https://play.google.com/store/apps/details?id=com.withings.thermo&hl=en) app has no feature to record journal/diary.

TODOs:
- [ ] Look for diary apps (not necessarily Withings compatible any more)

**Comment by 130s** [comment]
Not focused on health journal, but I just started using [Journey](https://journey.cloud/me) that provides both www and cellphone I/F.

**Comment by 130s** [comment]
> Not focused on health journal, but I just started using [Journey](https://journey.cloud/me) that provides both www and cellphone I/F.

After almost 4 years I switched from journey.cloud to GitHub https://github.com/130s/essay_in_idleness/issues/11
